### PR TITLE
Prepare for v0.18.0-pre.11

### PR DESCRIPTION
**Pull request by justinfagnani**
- [x] CHANGELOG.md has been updated

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## Upcoming
 
+## v0.18.0-pre.11 [02-24-2017]
+
+- Updated polymer-analyzer, adding nitial support for mixins and namespaces to the analye command.
```
Comment (abdonrd) [inline CHANGELOG.md]: analye > analyze :)

Comment (tony19) [inline CHANGELOG.md]: nitial > initial :)

**Comment by rictic** [comment]
(I just smashed Justin's commits)

**Comment by rictic** [comment]
Ran into some issues that necessitating removing the linter call on most of the integration tests. Documented here: https://github.com/Polymer/polymer-cli/issues/585

Probably worth going through review for these, just to get eyes on the issues.

**Comment by stramel** [comment]
Should we bump `polymer-build` to `^0.8.4`?

**Comment by rictic** [comment]
This landed as https://github.com/Polymer/polymer-cli/pull/595

**Comment by stramel** [comment]
:+1: Thanks @rictic and @justinfagnani!

**Pull request closed without merging**